setup.py

```diff
@@ -56,7 +56,7 @@
         "pybids==0.6.4",
         "setuptools>=40.0",
         "scikit-image",
-        "networkx==1.9",
+        "networkx",
```

Comment:
Only one comment -- have we checked to make sure the 1.9 networkx code in `gen_graph.py` is compatible with this?

Reply:
No, will check that now. Good catch @loftusa

Reply:
yep, works fine with networkx==2.2